### ISSUE TITLE
Suggestion for alternative methods to run ssh-pageant

### ISSUE THREAD

**Issue by gene-pavlovsky** [cuviper/ssh-pageant#62]
Using the README-recommended way to run ssh-pageant (via bash_profile), I was running into some problems.
I run Cygwin bash under a multi-tab console manager called ConEmu, and when I was exiting the shell, the tab wouldn't close, because the ssh-pageant process was still running in the background. This was rather annoying. 
Additionally, I was not able to use ssh or git when running those commands under cmd (or Far manager's internal command interpreter).

I've come up with [a different way](https://github.com/Maximus5/ConEmu/issues/2402#issuecomment-991626825) to start and use ssh-pageant, which solved all of these problems.

In short, a logon script is set up via local group policy editor, this logon script runs ssh-pageant, and sets the SSH_AUTH_SOCK user environment variable. Subsequent processes (ConEmu, Far, Cygwin bash, cmd etc) get this SSH_AUTH_SOCK in their environment, and don't need to run `ssh-pagent -r ...`.
